src/NetworkMap/Interfaces/View.elm

```diff
@@ -0,0 +1,167 @@
+module NetworkMap.Interfaces.View exposing (..)
+
+{-| Copyright 2020 Fluence Labs Limited
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-}
+
+import Array exposing (Array)
+import Dict exposing (Dict)
+import Element exposing (Element, alignRight, centerX, column, el, fillPortion, padding, paddingXY, row, spacing, text, width)
+import Element.Border as Border
+import Element.Font as Font
+import Element.Input as Input
+import Ions.Background as Background
+import Ions.Border as B
+import Ions.Font as F
+import Ions.Size as S
+import NetworkMap.Interfaces.Model exposing (Function, Inputs, Interface, Model, Module, Results)
+import NetworkMap.Interfaces.Msg exposing (Msg(..))
+import Palette exposing (accentButton, fillWidth, letterSpacing, limitLayoutWidth)
+import Screen.Model as Screen
+
+
+view : Screen.Model -> Model -> Element Msg
+view screen model =
+    let
+        modulesEl =
+            (model.interface
+                |> Maybe.map (interfaceForms model.id model.inputs model.results)
+            )
+                |> Maybe.withDefault Element.none
+    in
+    column [ fillWidth ]
+        [ modulesEl ]
+
+
+optionsView : Model -> Element Msg
+optionsView model =
+    let
+        btn =
+            Input.button
+                [ padding 10, Background.blackAlpha 60 ]
+            <|
+                { onPress = Just <| GetInterface model.id
+                , label =
+                    Element.el
+                        []
+                        (Element.text "Get Interfaces")
+                }
+    in
+    row [ limitLayoutWidth, Background.white, centerX ]
+        [ el [ alignRight, padding 5 ] <| btn
+        ]
+
+
+interfaceForms : String -> Inputs -> Results -> Interface -> Element Msg
+interfaceForms id inputs results interface =
+    let
+        modules =
+            interface.modules
+    in
+    column [ fillWidth, spacing 10 ] (Dict.values (modules |> Dict.map (\n -> \m -> moduleForms id n inputs results m)))
+
+
+defn : String -> Element Msg
```

Comment:
These names are nice when they're inside a `let` block and close to call contexts, but when detached, it's very unclear.